web/src/App.css

```
@@ -32,10 +33,64 @@
   }
 }
 
+.App {
+  background-color: white;
+  color: black;
+}
+
 .card {
   padding: 2em;
 }
 
 .read-the-docs {
   color: #888;
 }
+
+.title-card {
+  display: 
+}
+
+.project-title {
+  font: inter;
+  font-weight: bold;
+}
+
+.tagline {
+  font: inter;
```

Make sure your fonts have fallbacks, try this instead
```suggestion
  font: "Inter", sans-serif;
```

This will look for the "Inter" font that you specify in the head of the html document

---

Make sure to adjust all the other "Inter" font specifications too